### ISSUE TITLE
Could you please provide instructions for importing

### ISSUE THREAD

**Issue by liquidvisual** [Simonwep/selection#14]
Hi Simon, great library - I just can't seem to get it working. Since I can't import from node_modules (there's nothing in your libs folder) I've manually coped your code over to my project, but all attempts to import 'Selection' aren't working for me.

**Could you please add further instructions to your docs?** This is what I've tried (btw I'm new to this):

```
// import { Selection } from './selection';
// import Selection from './selection';
// import './selection';
// import * from '/selection';
// import '/selection/index.js/';
```

But I always get "cannot read property 'create' of undefined" or similar when I run ```Selection.create(options)```. 

Thanks!

**Comment by Simonwep** [comment]
### Without node
If you didn't use node you can use `<script src="https://cdn.jsdelivr.net/npm/@simonwep/selection-js@0.0.10/selection.min.js"></script>` to include it. Then you can use (in your script file)
`const selection = Selection.create` to create an instance.

### With node
With node you can use: 
`const selection = require('@simonwep/selection-js');`
If you are using [babel](https://babeljs.io/) you can use:
`import * as selection from '@simonwep/selection-js';`

**Comment by liquidvisual** [comment]
Thanks for the info, I'm currently using a Webpack setup from the Vue.js CLI. I've tried various combinations of the node methods but I'm not having much luck unfortunately:

```
// const selection = require('@simonwep/selection-js');
// import * as selection from '@simonwep/selection-js';
// import * as selection from './selection-js'; // using dist in the same directory

// Selection.create(options);
// selection.create(options);
```

I still keep getting ```"TypeError: Selection.create is not a function"``` 

Is there something I could be missing?

**Comment by Simonwep** [comment]
Okay, I've made some really big mistakes... I have released an new version 0.11, now everything should work. If not, you could send your dependencies and I will look wheres the problem.

**Comment by liquidvisual** [comment]
All good, no need to apologise :) I'm so very grateful for your work and time. The latest **0.11** seems to have removed that error I was getting - (woo!) but I'm now getting a different error when I click and drag:

```
Uncaught TypeError: Cannot read property 'getBoundingClientRect' of undefined
```

This could very well be my own error, since I'm new to this, but I'd really appreciate any assistance you'd be willing to give :) 

1. This is the [live preview](https://5b1164d0e39e7c0f66f2806a--kleptobreaker.netlify.com/start) of what I'm building. _(Please be patient though, it takes a while to load)_.

2. This is the [component](https://github.com/liquidvisual/klepto-1017/blob/filesystem-selection.js/src/components/KleptOS/System/Apps/AppFileSystem/IconView/IconGrid.vue) that controls the grid/selection.

3. This is the [branch](https://github.com/liquidvisual/klepto-1017/tree/filesystem-selection.js)

4. Finally, this is a **working** [Codepen](https://codepen.io/liquidvisual/pen/NzqzxN) of what I'm trying to implement into my Vue component.

Apologies if that's a bit overwhelming, please let me know if I can compensate you for your time.

**Comment by Simonwep** [comment]
No problem :)
The project should work, so every discovered issue helps me!
I will take a look tomorrow because I'm currently really busy :/

**Comment by liquidvisual** [comment]
Thanks so much! I really appreciate it :)

**Comment by Simonwep** [comment]
Okay, I cannot exactly identify the problem but is maybe here:
```javascript
// Query selectors from elements from which the siblings can be selected
containers: ['.icon-grid'],

// Query selectors from elements which can be selected
selectables: ['.icon-grid-square'],
```

Try to use only on of them, if you say `containers: ['.icon-grid']` it will use **all** direct siblings, but you also add `selectables: ['.icon-grid-square']` which would **also** (but directly) select all those elements wich you want. So maybe remove one of these properties.

**Comment by liquidvisual** [comment]
Thanks for looking at it. I tried your suggestion but I couldn't solve it.

I did manage to stumble on a potential lead though, when I remove the import and instead add the CDN link to my index.html

```
<script src="https://cdn.jsdelivr.net/npm/@simonwep/selection-js/selection.min.js"></script>
```

Then the grid squares actually highlight when I click them. Drag selecting still doesn't work, but I can at least see the **'.selection-area'** in the DOM toggling its display:

```
<div class="selection-area" style="top: 0px; left: 0px; position: fixed; display: none;"</div>
```

When I do the import, the **'.selection-area'** appears before the closing body with no class, and doesn't toggle display:

```
<div style="top: 0px; left: 0px; position: fixed;"></div>
```

**Comment by liquidvisual** [comment]
I'm going to try and create a stripped down npm test scenario in Codepen to see where the problem is. It's all getting a bit confusing! I'll report back once i've set it up.

**Comment by Simonwep** [comment]
Thanks a lot for trying it, it really helps me :)
But the first CDN link shouldn't work, you would include an very, very old version where I kept my build files in the root directory, so it's actually important to use the link to which contains the /dist.

**Comment by liquidvisual** [comment]
I finally fixed it! Woohoo! Your code is perfectly fine, it was an oversight on my end :)

In the process of setting up my environment, I discovered an awesome online editor (silver lining?) - I'll share the example with you below. It works great with NPM now. The main issue I was having was due to my html/body being zero height. I completely overlooked it but had to ravage my whole application to find it. 

Very excited to move on now, your excellent library is a such an integral part of my application. 👍👍

https://codesandbox.io/s/rlx7w7l76p

Thanks for all the help!
